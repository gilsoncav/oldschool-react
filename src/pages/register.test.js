import Register from './register';

describe('Register page component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Register />);
    expect(wrapper).toMatchSnapshot();
  });
});
