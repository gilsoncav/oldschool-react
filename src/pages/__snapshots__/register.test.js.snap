// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register page component renders correctly 1`] = `
<Row
  gutter={0}
>
  <Col
    offset={9}
    span={6}
    style={
      Object {
        "marginTop": "10%",
      }
    }
  >
    <HomeCard>
      <withRouter(Form(RegisterForm)) />
    </HomeCard>
  </Col>
</Row>
`;
